const.go

```diff
@@ -2,6 +2,9 @@ package notary
 
 // application wide constants
 const (
+	// MaxMetaSize is the maximum size of metadata - 5MiB.
```

Comment:
Should be clear it's only the max size for a timestamp, which we download without any knowledge about exactly how large it will be.